nms/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js

```diff
@@ -1065,14 +1101,19 @@ export function validateSubscriberInfo(
   }
   return '';
 }
+
 type SubscriberError = $Values<typeof SUBSCRIBER_ADD_ERRORS>;
+
 /**
- * Check subscriber fields format
- * Returns array of errors
+ * Checks subscriber fields format
  *
  * @param {Array<SubscriberInfo>} subscribers Array of subcribers to validate
+ * @returns {Array<string>} Returns array of errors
  */
-export function validateSubscribers(subscribers: Array<SubscriberInfo>) {
+export function validateSubscribers(
+  subscribers: Array<SubscriberInfo>,
+  isDelete?: boolean,
```

Comment:
Rather than using this `isDelete` parameter for so many functions, stick to a required `SubscriberActionType`. It's clearer.